src/main/kotlin/net/minestom/dependencies/DependencyResolver.kt

```diff
@@ -16,19 +17,4 @@ interface DependencyResolver {
      */
     @Throws(UnresolvedDependencyException::class)
     fun resolve(id: String, targetFolder: File): ResolvedDependency
-}
-
-/**
- * Resolved Dependency.
- * Holds its coordinates (group, artifact, version), which are allowed to be empty if needed
- *
- * The contentsLocation URL represents the location of the dependency, on local storage.
- */
-data class ResolvedDependency(val group: String, val name: String, val version: String, val contentsLocation: URL, val subdependencies: List<ResolvedDependency>) {
-    fun printTree(indent: String = "") {
-        println("$indent- $group:$name:$version ($contentsLocation)")
-        for (dep in subdependencies) {
-            dep.printTree("$indent  ")
-        }
-    }
-}
```

Comment:
Moved into new file. If there should be one class per file, they are written in the PascalCase, otherwise usually in the lowerCamelCase.